<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>540</width>
                <height>252</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Products</string>
        </property>
        <widget class="QTableWidget" name="tableProduct">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>30</y>
                    <width>491</width>
                    <height>192</height>
                </rect>
            </property>
            <property name="columnCount">
                <number>3</number>
            </property>
        </widget>
    </widget>
</ui>